tutor/src/components/reference-book/reference-book.cjsx

```diff
@@ -7,7 +7,7 @@ classnames = require 'classnames'
 
 NavBar = require './navbar'
 Menu = require './slide-out-menu'
-{ChapterSectionMixin, ResizeListenerMixin} = require 'shared'
+{ResizeListenerMixin} = require 'shared'
 PageShell = require './page-shell'
 PageNavigation = require './page-navigation'
```

Comment:
hmm, i think it would be more clear here to use this as `ReferenceViewPageNavigation` vs `PageNavigation`